### PR TITLE
[Angular] Fix typescript errors

### PR DESCRIPTION
**Pull request by illiakovalenko**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update (non-breaking change; modified files are limited to the `/docs` directory)

### CODE REVIEW

File: samples/angular/scripts/disconnected-mode-proxy.ts [hunk 1]
```diff
@@ -25,6 +25,8 @@ const proxyOptions = {
   appName: config.appName,
   watchPaths: ['./data'],
   language: config.language,
+  // Additional transpilation is not needed
+  requireArg: '',
```
Comment (CobyPear) [inline samples/angular/scripts/disconnected-mode-proxy.ts:29]: This makes sense, but would it be cleaner to pass in  `'none'` 
or something other than an empty string, and handle it inside of the `ManifestManager`?

Something like the following to go [here](https://github.com/Sitecore/jss/blob/19954b1e5a82e0f267fbfb3434f12c3271d8359e/packages/sitecore-jss-dev-tools/src/manifest-manager.ts#L72)?
```
if (requireArg === 'none' || requireArg === false) {
    this.requireArg = '';
}
```

**Review by CobyPear — COMMENTED** [review]
One question/suggestion but looks good.

File: samples/angular/scripts/disconnected-mode-proxy.ts (hunk 1 above)
Comment (illiakovalenko) [reply samples/angular/scripts/disconnected-mode-proxy.ts]: Good idea, resolved

**Review by CobyPear — APPROVED** [review]
Looks good, approved!

**Pull request merged**